contracts/strategies/CoreBaseStrategy.sol

```diff
@@ -107,6 +108,16 @@ abstract contract CoreBaseStrategy is IBaseStrategy, Transfer {
         onlyPoolManager(msg.sender)
     {
         _beforeWithdraw(_token, _amount, _recipient);
+        // If the token is the pool token, revert if the amount is greater than the pool amount
```

Comment:
In case the strategy is keeping donation tokens in the strategy then the pool managers will be able to withdraw those donation tokens, this is important to note.

Reply:
Yeah this makes sense for being the default behaviour